src/main/rx/ghst.c

```diff
@@ -191,6 +198,10 @@ STATIC_UNIT_TESTED uint8_t ghstFrameStatus(rxRuntimeState_t *rxRuntimeState)
             return RX_FRAME_COMPLETE | RX_FRAME_PROCESSING_REQUIRED;            // request callback through ghstProcessFrame to do the decoding  work
         }
 
+        if(crc != ghstValidatedFrame.bytes[fullFrameLength - 1] ) {
+            DEBUG_INCR(DEBUG_GHST, DEBUG_GHST_CRC_ERRORS);
```

Comment:
This is somewhat risky. There is nothing that would prevent some other function from setting the debug values - either intentionally or by mistake. Generally not a good idea to use the `debug` globals for persistent storage.

Reply:
I wouldn't exactly call it persistent storage. It has been added to help test pilots debug issues potentially caused by the lack of UART error checking in the BF drivers.
If another source decided to overwrite these values it wouldn't be the end of the world.
Besides, this is exactly what these debug values are for, to let me get data on the OSD, black box, and real time sensor logging for bench tests, all without bloating the code with special cases.

Reply:
Keep a local counter and set the debug value to that. This is how all the debugging code works throughout Betaflight. We don't rely on the `debug` array for persistent storage.